src/message/action.js

```diff
@@ -879,8 +879,10 @@ function QueueReset() {
  * @property {String} Interface Interface
  * @augments Action
  */
-function QueueRemove() {
+function QueueRemove(queue, asteriskInterface) {
```

Comment:
Oh I see your point. I was actually refering to keeping consistency inside nami itself by using the same arguments in the same order. Makes sense?